precompile/living-assets/src/lib.rs

```diff
@@ -49,10 +49,18 @@ where
 
 		match selector {
 			Action::CreateCollection => {
+				let mut input = handle.read_input()?;
+				input.expect_arguments(1)?;
+
+				let base_uri: Vec<u8> = match input.read::<Bytes>() {
```

Comment:
I added a test to expose the behavior ( https://github.com/freeverseio/laos-ownership-node/pull/110/commits/51342a0bcb16f4bbdf0ec3ce293960325e0223f4 ). However it will error if the arguments are less than required.